### ISSUE TITLE
Kolibri Android App - 'Webpage not available' message is displayed after launching the app

### ISSUE THREAD

**Issue by pcenov** [learningequality/kolibri#8921]
## Observed behavior
I'm seeing a 'Webpage not available' message after launching the Android app which disappears after 15-20 seconds and then I can interact normally with the app.

## Steps to reproduce the issue
1. Install the following Android build: https://buildkite.com/learningequality/kolibri-android-installer/builds/3613
2. Launch the app
3. Observe that an error screen is displayed and only after waiting about 15-20 seconds I can actually see the app fully loaded

## Additional information
![Screenshot_20211214-165753_Kolibri](https://user-images.githubusercontent.com/79847249/146022948-46acc406-35c7-4fa8-b27e-b24bc8095b99.jpg)

## Logs and DB files: 
[AndroidLogsAndDB.zip](https://github.com/learningequality/kolibri/files/7712473/AndroidLogsAndDB.zip)

## Usage Details
 - OS: Android 7, Samsung S7

**Comment by pcenov** [comment]
@radinamatic

**Comment by rtibbles** [comment]
Unfortunately this is currently expected behaviour due to an Android API upgrade we had to do. I will transfer this issue to the android installer repository, but note that we won't fix this prior to releasing 0.15.
